test/base/utils/CommonTestUtils.scala

```diff
@@ -17,6 +17,7 @@
 package base.utils
 
 import scala.util.Random
+import scala.language.postfixOps
```

Comment:
All postfix with this scala version have beed disabled. Here I have imported for this only.